### ISSUE TITLE
etcd version 3 does not send cfn-signal

### ISSUE THREAD

**Issue by jeremyd** [kubernetes-incubator/kube-aws#497]
<EDITED, this happens also with etcd2>  I tried the new etcd support and specified version: 3.1.5 in the cluster.yaml.  However, etcd starts, but is version 3.0.x and also cfn-signal never fires.  I attempted to debug why it does not fire since etcd3 was able to startup ok and I'm seeing a weird issue.  It does not look like cfn-signal was ever triggered to run and when I manually do systemctl start cfn-signal it hangs forever.  When I look at the processlist for some reason it's showing this systemd-tty-ask-password-agent running...  Using latest master.

```
root      3710  0.0  0.0  33420  3084 pts/0    S+   18:25   0:00                      \_ systemctl start cfn-signal
root      3711  0.0  0.0  33288  3188 pts/0    S+   18:25   0:00                          \_ /usr/bin/systemd-tty-ask-password-agent --watch
```

I just tried etcd2 also and it did not receive the signal, however this problem is slightly different since the cfn-signal did startup but cloudformation did not see the signal come in.

**Comment by camilb** [comment]
I'm still investigating this issue. It's weird because it worked once, 3 ETCD nodes, all sent cfn-signal. Even got a snapshot on a S3 bucket .  Now I don't manage to pass even one "success".Tomorrow  I'm planning to increase the waiting time for the signal and try debugging each failed service on ETCD nodes.

**Comment by camilb** [comment]
@jeremyd maybe #504 fix this issue, in case you had terminated instances from previous deployments, as @mumoshu pointed this bug in #501.

**Comment by mumoshu** [comment]
Thanks to both of you :bow:
#504 is merged and kube-aws v0.9.6-rc.1 which includes the fix is released.
@jeremyd Could you please confirm if the new release work for you now?

**Comment by jeremyd** [comment]
This works now, thanks!
